nodebuilder/blob/cmd/blob.go

```diff
@@ -41,14 +41,28 @@ func init() {
 		&fee,
 		"fee",
 		-1,
-		"specifies fee for blob submission",
+		"specifies fee(in TIA) for blob submission",
 	)
 
 	submitCmd.PersistentFlags().Uint64Var(
 		&gasLimit,
 		"gas.limit",
 		0,
-		"specifies max gas for the blob submission",
+		"sets the maximum amount of gas that is allowed to consume during blob submission",
+	)
+
+	submitCmd.SetUsageTemplate("" +
+		"celestia blob submit [namespace] [blobData] [flags]\n\n " +
+		"Flags:\n" +
+		"      --fee       int      specifies fee(in TIA) for blob submission(optional)\n" +
+		"      --gas.limit uint64   sets the maximum amount of that gas is allowed " +
+		"to consume during blob submission(optional)\n" +
+		"      -h, --help           help for submit\n" +
+		"NOTE: fee and gas.limit params will be calculated automatically if they are not provided as arguments.\n\n" +
+		"Global Flags:\n" +
+		"      --token string    Authorization token (if not provided, " +
+		"the CELESTIA_NODE_AUTH_TOKEN environment variable will be used)\n" +
+		"      --url   string    Request URL (default \"http://localhost:26658\")",
```

Comment:
This sucks to have hardcoded like this in string - can we not add on usage options to the flags themselves?

Reply:
when you add flags to a specific cmd, it should spit out the flags for you

Reply:
cmd.Flag("flagKey") returns the flag but anyway we will have to hardcode the string if we want to hide unneeded information

Reply:
reworked